src/Common/src/System/Net/HttpKnownHeaderNames.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    internal static class HttpKnownHeaderNames
```

Comment:
At this point in time, the API surface for this contract is locked for this initial port.  This contract is to support the System.Net.Requests contract which is the older HTTP APIs (HttpWebRequest).  However, perhaps this might be considered for a future revision of the System.Net.Http contract.